tests/common.py

```diff
@@ -132,6 +132,57 @@ def wait_for_model_evaluated(stub, metadata, model_id, model_version_id):
     # At this point, the model has successfully finished evaluation.
 
 
+def wait_for_dataset_version_ready(stub, metadata, dataset_id, dataset_version_id):
+    while True:
```

Comment:
no timeout?

Reply:
Nope, copy-pasted this solution from the existing model waiting functions. All tests run in parallel anyway and have a global timeout to make sure we do not get stuck waiting forever.